test/update-db.test.js

```diff
@@ -60,126 +97,105 @@ it('throws on missing lockfile', async () => {
   )
 })
 
-it('updates caniuse-lite for npm', async () => {
-  let dir = await chdir('update-npm', 'package.json', 'package-lock.json')
+it('shows target browser changes', async () => {
+  let dir = await chdir('browserslist-diff',
+    'package.json', 'package-lock.json')
 
-  expect(runUpdate()).toContain(
-    'Current version: 1.0.30001030\n' +
-    `New version:     ${ caniuse.version }\n` +
-    'Removing old caniuse-lite from lock file\n' +
-    'Installing new caniuse-lite version\n' +
-    '$ npm install caniuse-lite\n' +
-    'Cleaning package.json dependencies from caniuse-lite\n' +
-    '$ npm uninstall caniuse-lite\n' +
-    'caniuse-lite has been successfully updated\n'
+  expect(runUpdate()).toMatch(
+    // eslint-disable-next-line max-len
+    /(Target browser changes:\n([+-] \w+ [\d.-]+\n)+)|(No target browser changes)/
   )
 
   let lock = JSON.parse(await readFile(join(dir, 'package-lock.json')))
   expect(lock.dependencies['caniuse-lite'].version).toEqual(caniuse.version)
 })
 
+it('shows error when browsers list can\'t be retrieved', async () => {
+  let dir = await chdir('browserslist-diff-error',
+    'package.json', 'package-lock.json')
+
+  expect(runUpdate())
+    .toContain(
+      'Problem with browsers list retrieval.\n' +
+      'Target browser changes won’t be shown.\n'
+    )
+
+  let lock = JSON.parse(await readFile(join(dir, 'package-lock.json')))
+  expect(lock.dependencies['caniuse-lite'].version).toEqual(caniuse.version)
+})
+
 it('updates caniuse-lite without previous version', async () => {
   let dir = await chdir('update-missing', 'package.json', 'package-lock.json')
+  checkRunUpdateContents('none', 'npm')
+  let lock = JSON.parse(await readFile(join(dir, 'package-lock.json')))
+  expect(lock.dependencies['caniuse-lite']).toBeUndefined()
+})
 
-  expect(runUpdate()).toContain(
-    `New version:     ${ caniuse.version }\n` +
-    'Removing old caniuse-lite from lock file\n' +
-    'Installing new caniuse-lite version\n' +
-    '$ npm install caniuse-lite\n' +
-    'Cleaning package.json dependencies from caniuse-lite\n' +
-    '$ npm uninstall caniuse-lite\n' +
-    'caniuse-lite has been successfully updated\n'
-  )
+it('updates caniuse-lite for npm', async () => {
+  let dir = await chdir('update-npm', 'package.json', 'package-lock.json')
+  checkRunUpdateContents('1.0.30001030', 'npm')
 
   let lock = JSON.parse(await readFile(join(dir, 'package-lock.json')))
-  expect(lock.dependencies['caniuse-lite']).toBeUndefined()
+  expect(lock.dependencies['caniuse-lite'].version).toEqual(caniuse.version)
+})
+
+it('skips the npm update if caniuse-lite is up to date', async () => {
+  let dir = await chdir('update-npm', 'package.json', 'package-lock.json')
+  // update the dir to the current version
```

Comment:
Can you remove comments? I prefer to change function names if the code is not readable without comments.